fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/aggregate/AggregateAccumulators.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * AggregateAccumulators.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.cursors.aggregate;
+
+import com.apple.foundationdb.record.query.predicates.Value;
+
+import javax.annotation.Nonnull;
+
+/**
+ * A class that contains static initializers for various aggregate accumulators.
+ */
+public class AggregateAccumulators {
```

Comment:
`COUNT(*)` and `COUNT(x)` (meaning count-not-null) also introduce interesting cases.

Reply:
This will be included (see design document) in a subsequent PR (that includes Count)